e2e-tests/commands/insurance/complete-and-submit-different-name-on-policy-form.js

```diff
@@ -1,29 +1,23 @@
-import { INSURANCE_FIELD_IDS } from '../../constants/field-ids/insurance';
-import mockApplication from '../../fixtures/application';
-
-const {
-  ACCOUNT: {
-    FIRST_NAME,
-    LAST_NAME,
-    EMAIL,
-  },
-} = INSURANCE_FIELD_IDS;
-
-const { POLICY_CONTACT } = mockApplication;
-
 /**
  * completeAndSubmitDifferentNameOnPolicyForm
- * Runs through the different name on policy form in the "policy" section
- * @param {String} First name
- * @param {String} Last name
- * @param {String} email
+ * Complete and submit the "different name on policy form"
+ * @param {String} firstName: First name
+ * @param {String} lastName: Last name
+ * @param {String} email: Email
+ * @param {String} position: Position
  */
 const completeAndSubmitDifferentNameOnPolicyForm = ({
-  firstName = POLICY_CONTACT[FIRST_NAME],
-  lastName = POLICY_CONTACT[LAST_NAME],
-  email = POLICY_CONTACT[EMAIL],
+  firstName,
```

Comment:
could we maybe set the default different name here? so we don't have to manually set it in the change your answers section and the policy section tests?

Reply:
The default params are now set in `completeDifferentNameOnPolicyForm`, along with conditionally entering values.

Manually setting the values to false or empty strings in the tests are intentional, so that only specific fields are entered. We have this pattern in other places